extensions/wikia/GlobalWatchlist/GlobalWatchlistTask.class.php

```diff
@@ -43,34 +43,50 @@ public function addWatchers( $databaseKey, $nameSpace, array $watchers ) {
 
 			// Skip revisions that doesn't exist
 			if ( !empty( $revision ) ) {
+				$watchersToAdd = [];
 				$globalWatchlistBot = new GlobalWatchlistBot();
-
-				$db = wfGetDB( DB_MASTER, [ ], \F::app()->wg->ExternalDatawareDB );
 				foreach ( $watchers as $watcherID ) {
 					if ( $globalWatchlistBot->shouldNotSendDigest( $watcherID ) ) {
-						$this->clearGlobalWatchlistAll( $watcherID );
```

Comment:
Doing this `clearGlobalWatchlistAll()` here actually isn't necessary. When we run the `sendWeeklyDigest.php` script (which runs once a week), we'll call `clearGlobalWatchlistAll()` for every user in the db. So removing it from here to simplify.

See [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/GlobalWatchlist/GlobalWatchlistBot.class.php#L13-L18).